docs/source/R-api.rst

```diff
@@ -1309,13 +1309,16 @@ to be used by package authors to extend the supported MLflow models.
 Arguments
 ---------
 
-========= ===================================================================
-Argument  Description
-========= ===================================================================
-``model`` The loaded MLflow model flavor.
-``data``  A data frame to perform scoring.
-``...``   Optional additional arguments passed to underlying predict methods.
-========= ===================================================================
++-----------+---------------------------------------------------------+
+| Argument  | Description                                             |
++===========+=========================================================+
+| ``model`` | The loaded MLflow model flavor.                         |
++-----------+---------------------------------------------------------+
+| ``data``  | A data frame to perform scoring.                        |
++-----------+---------------------------------------------------------+
+| ``...``   | Optional additional arguments passed to underlying      |
+|           | predict methods.                                        |
++-----------+---------------------------------------------------------+
```

Comment:
I updated the version of pandoc to `2.16.2` from `2.7.1`, which added some changes in this file.